ext/drreg/drreg.c

```diff
@@ -659,7 +1052,13 @@ drreg_forward_analysis(void *drcontext, instr_t *start)
         pt->reg[GPR_IDX(reg)].app_uses = 0;
         drvector_set_entry(&pt->reg[GPR_IDX(reg)].live, 0, REG_UNKNOWN);
     }
-
+#ifdef X86
+    for (reg = DR_REG_APPLICABLE_START_SIMD; reg <= DR_REG_APPLICABLE_STOP_SIMD; reg++) {
+        pt->simd_reg[SIMD_IDX(reg)].app_uses = 0;
+        drvector_set_entry(&pt->simd_reg[SIMD_IDX(reg)].live, 0, SIMD_UNKNOWN);
+        pt->simd_reg[SIMD_IDX(reg)].ever_spilled = false;
```

Comment:
This does not match the GPR code.  Is one of them incorrect?  This seems familiar: did we already have a discussion on this in the past?

Reply:
You're referring to [i#3827](https://github.com/DynamoRIO/dynamorio/issues/3827)
I've added a comment.

Reply:
Resolving, re-open if you think the comment does not suffice right now.